.buildkite/scripts/packaging/packaging.sh

```diff
@@ -12,8 +12,10 @@ set -ueo pipefail
 
 BEAT_DIR=${1:?-"Error: Beat directory must be specified."}
 
-#Use newer multiarch support for packaging
-docker run --privileged --rm tonistiigi/binfmt:master --install all
+BINFMT_IMAGE="tonistiigi/binfmt:qemu-v8.1.5"
```

Comment:
@kruskall could we please also add the comment from @mauri870 

```
+# Install emulators for multi-arch builds
+#
+# NOTE: Pinned to QEMU v8.x since v9.x breaks compilation of amd64 binaries in arm64.
+# See: https://gitlab.com/qemu-project/qemu/-/issues/2560
```

as we'd benefit from context here